keyboards/handwired/dygma/raise/matrix.c

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2018-2021 James Laird-Wah, Islam Sharabash
```

Comment:
Could this be converted to matrix lite? 
https://docs.qmk.fm/#/custom_matrix?id=lite

Reply:
I don't think so, we want to avoid debouncing here since that's already done in the dedicated keyscanner MCUs built into each split.

Reply:
Well, setting debounce to 0 should handle that (eg, disable bouncing). 

But in general, matrix lite is te way we want to go, since it reduces complexity (eg, don't need the full matrix and all the misc code with it)

Reply:
Is changing to matrix lite a blocker?

Reply:
No, it's not.

Reply:
To clarify, I don't think it should be a blocker, but it should be done anyways.  It's not just debounce, but it is that it simplifies and handles a bunch of the minutiae, too. 

And for the debounce, you can set the debounce time to "custom", and just add empty functions, basically, so that no real debouncing actually happens.